src/Command/ParallelProcessQueueCommand.php

```diff
@@ -68,5 +68,10 @@ protected function doFetchItems(InputInterface $input, ?OutputInterface $output)
     protected function runSingleCommand(string $item, InputInterface $input, OutputInterface $output): void
     {
         $this->importProcessingService->processQueueItem((int) $item);
+
+        // call the garbage collector if memory consumption is > 100MB
+        if (memory_get_usage() > 100000000) {
```

Comment:
is that really necessary? 
actually, that should already be done by https://github.com/pimcore/pimcore/blob/11.x/lib/Console/Traits/Parallelization.php#L61

Reply:
maybe we should consider batch sizes somehow, when there is no `\Webmozarts\Console\Parallelization` in here: https://github.com/pimcore/pimcore/blob/11.x/lib/Console/Traits/ParallelizationBase.php#L44

Reply:
Reverted. please see https://github.com/pimcore/pimcore/issues/16065